.eslintrc.js

```diff
@@ -1114,10 +1114,18 @@ module.exports = {
      * Enterprise Search overrides
      */
     {
+      // All files
       files: ['x-pack/plugins/enterprise_search/**/*.{ts,tsx}'],
-      excludedFiles: ['x-pack/plugins/enterprise_search/**/*.{test,mock}.{ts,tsx}'],
       rules: {
         'react-hooks/exhaustive-deps': 'off',
+        'react/jsx-boolean-value': ['error', 'never'],
+      },
+    },
+    {
+      // Source files only - allow `any` in test/mock files
```

Comment:
I split our overrides up into 2 sections so that we could have rules (e.g., particularly the unnecessary backticks one coming up) that applied to all files, and rules that applied to just source files.

That being said, I'm actually wondering if we should only enforce `never` for source code files and allow devs to specify `someBool={true}` in test files. This provides a greater contrast and could help understanding for certain props, e.g.

```js
it('does something when someBool is false', () => {
  const wrapper= shallow(<SomeComponent someBool={false} />);
  // ...
});

it('does something else when someBool is true', () => {
  const wrapper= shallow(<SomeComponent someBool={true} />);
  // ...
});
```

What do y'all think?

Reply:
I don't have a preference.

Reply:
I don't have a preference.

Reply:
It should be consistent between the two, I'd rather have `someFlag={true}` universally than these different

Reply:
Works for me, I'm good either way